src/kernel/serial.zig

```diff
@@ -0,0 +1,39 @@
+const arch = @import("arch.zig").internals;
```

Comment:
I believe there is a interrupt handler for the serial as well, IRQ 3 and 4. U can use my PIC interface to get the IRQ const

Reply:
Adding the interrupt handlers is only really used for receiving serial input so that could wait until we add that.

Reply:
No problem, was looking as the osdev page on serial, and other code snipits and had interrupt stuff